bugfix: textarea char tails clipped

tmikeschu opened the pull request:
[Gitlab issue screenshots and notes](https://gitlab.com/care-exchange/care-exchange-application/-/issues/145)

Fixes clipped character tails in textarea fields.

tmikeschu committed:
bugfix: textarea char tails clipped

jasonbrandt42 commented on src/app/prompts/prompts/prompts.component.html:
```diff
@@ -1,213 +1,263 @@
+        <div fxLayout="row" fxLayoutAlign="space-between center " *ngFor="let share of shares; let i = index">
+          <p class="cardsizeleft">
+            <strong
+              >{{ share.item }} <span *ngIf="share.sizes != null">({{ share.size }})</span></strong
```
Again, these are really odd breaks.  Can we keep tags mostly on same line.

tmikeschu committed:
Revert "bugfix: textarea char tails clipped"

This reverts commit 3dcec29fca2f551a7073cf823d7f1fd3afc8dc96.

tmikeschu committed:
textarea height set

tmikeschu committed:
textarea height set

tmikeschu commented:
@jasonbrandt42 also removed all the prettier business from this. moved the notes label within a `mat-label` tag to get the styling to work as designed by material.
